requirements.txt

```diff
@@ -21,33 +21,34 @@ django-taggit==0.14.0
 django-xmlrpc==0.1.5
 easy-thumbnails==2.3
 ecdsa==0.13
--e git+https://github.com/dcramer/django-paypal.git#egg=django_paypal-master
+# -e git+https://github.com/dcramer/django-paypal.git#egg=django_paypal-master
+-e git+https://github.com/spookylukey/django-paypal.git#egg=django_paypal-master
 gunicorn==19.5.0
 html2text==2015.2.18
-html5lib==0.95
+html5lib==0.999
 httplib2==0.9
-lxml==3.4.2
+lxml>=3.4.2
 markdown2==2.3.5
 oauthlib==0.7.2
 paramiko==1.17.6
 Pillow==2.6.1
 polib==1.0.6
 Pygments==2.0.2
 PyJWT==0.4.1
-python-memcached==1.48
-python-oembed==0.2.1
+python-memcached>=1.48
+python-oembed
 python-openid==2.2.5
 python-social-auth==0.2.9
-python-twitter==2.2
+python-twitter==3.4.2
 pytz
-raven==5.1.1
+# raven==5.1.1
 requests==2.5.3
 responses==0.8.1
 requests-oauthlib==0.4.2
 simplejson==3.6.5
-six==1.9.0
-stripe==1.19.1
-suds==0.4
+six>1.9.0
+stripe
+suds-jurko
```

Comment:
what's suds-jurko?

Reply:
It is the Python3.7 compatible version of suds.

Reply:
It is used in assopy:

epcon ➤ rg -t py suds                                                                                                               git:dev/ep2019py37_unicode
misc/assopy-dist/setup.py
36:        'suds',

assopy/clients/vies.py
3:from suds.client import Client

Reply:
It was probably used long time ago, but it's not used right now, also I removed that whole veis "integration" few days ago -> https://github.com/EuroPython/epcon/pull/806